ArchiSteamFarm/Bot.cs

```diff
@@ -2846,7 +2846,7 @@ public sealed class Bot : IAsyncDisposable {
 				Utilities.InBackground(ArchiWebHandler.MarkInventory);
 			}
 
-			if (BotConfig.SendSetsOnCompleted && !BotConfig.SetTypesToComplete.IsEmpty) {
+			if (!BotConfig.CompleteTypesToSend.IsEmpty) {
```

Comment:
`IsEmpty` is not portable across different collections, check count against 0, it's `O(1)` anyway, and follows ASF convention.